src/System/include/BipedalLocomotion/System/ContactWrench.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file ContactWrench.h
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_SYSTEM_CONTACT_WRENCH_H
+#define BIPEDAL_LOCOMOTION_SYSTEM_CONTACT_WRENCH_H
+
+#include <iDynTree/Core/Wrench.h>
+#include <iDynTree/Model/Indices.h>
```

Comment:
Note that ``idyntree-model`` is linked privately.

Reply:
Fixed in a8f8df6fd4f2ab22e28ca491413d071d76a5a29c